TanukiKitTests/ProjectTests.swift

```diff
@@ -0,0 +1,157 @@
+        XCTAssertEqual(project.creatorID, 3)
+        XCTAssertEqual(project.namespace!.id, 3)
+        XCTAssertEqual(project.namespace!.name, "Diaspora")
+        XCTAssertEqual(project.namespace!.path, "diaspora")
+        XCTAssertEqual(project.namespace!.ownerID, 1)
+        XCTAssertEqual(project.namespace!.createdAt, TestHelper.parseDate(date: "2013-09-30T13:46:02Z"))
+        XCTAssertEqual(project.namespace!.updatedAt, TestHelper.parseDate(date: "2013-09-30T13:46:02Z"))
+        XCTAssertEqual(project.namespace!.namespaceDescription, "")
+        XCTAssertEqual(project.namespace!.avatar!.url, nil)
+        XCTAssertEqual(project.namespace!.shareWithGroupLocked, nil)
+        XCTAssertEqual(project.namespace!.visibilityLevel, nil)
+        XCTAssertEqual(project.namespace!.requestAccessEnabled, nil)
+        XCTAssertEqual(project.namespace!.deletedAt, nil)
+        XCTAssertEqual(project.namespace!.lfsEnabled, nil)
+        XCTAssertEqual(project.avatarURL, URL(string: "http://example.com/uploads/project/avatar/3/uploads/avatar.png"))
+        XCTAssertEqual(project.starCount, 0)
+        XCTAssertEqual(project.forksCount, 0)
+        XCTAssertEqual(project.openIssuesCount, 1)
+        XCTAssertEqual(project.visibilityLevel, VisibilityLevel.Private)
+        XCTAssertEqual(project.createdAt, TestHelper.parseDate(date: "2013-09-30T13:46:02Z"))
+        XCTAssertEqual(project.lastActivityAt, TestHelper.parseDate(date: "2013-09-30T13:46:02Z"))
+        XCTAssertEqual(project.lfsEnabled, nil)
+        XCTAssertEqual(project.runnersToken, "b8bc4a7a29eb76ea83cf79e4908c2b")
+        XCTAssertEqual(project.onlyAllowMergeIfBuildSucceeds, false)
+        XCTAssertEqual(project.requestAccessEnabled, false)
+        XCTAssertEqual(project.permissions!.projectAccess!.accessLevel, 10)
+        XCTAssertEqual(project.permissions!.projectAccess!.notificationLevel, 3)
+        XCTAssertEqual(project.permissions!.groupAccess!.accessLevel, 50)
+        XCTAssertEqual(project.permissions!.groupAccess!.notificationLevel, 3)
+    }
+
+    func testProjectEventsParsing() {
+        let event = Event(TestHelper.JSONFromFile(name: "Event") as! [String: AnyObject])
+        XCTAssertEqual(event.title, nil)
+        XCTAssertEqual(event.projectID, 37)
+        XCTAssertEqual(event.actionName, "pushed to")
+        XCTAssertEqual(event.targetID, nil)
+        XCTAssertEqual(event.targetType, nil)
+        XCTAssertEqual(event.authorID, 8)
+        XCTAssertEqual(event.data!.objectKind, "push")
+        XCTAssertEqual(event.data!.eventName, "push")
+        XCTAssertEqual(event.data!.before, "7a6a450ca607c9c0be539bb91c5c7eb6bdf4cc79")
+        XCTAssertEqual(event.data!.after, "bde8115a0d77c6a0bcad8e427ecbc38c4a6a4f5f")
+        XCTAssertEqual(event.data!.ref, "refs/heads/master")
+        print("DEBUG: \(event.data!.commits)")
+        XCTAssertEqual(event.data!.commits![0].message, "The private token is no longer given\n")
+        // TODO: Finish Parsing tests
```

Comment:
Oh I wasn't completely sure wether I should do those parsing tests as they just repeat the commit and event parsing tests? What would you prefer me to do?